cfgov/unprocessed/css/print.less

```diff
@@ -58,8 +58,10 @@
     }
 
     a[href="/"]:after,
-    a[href*="?authors"]:after {
-        // Don't append hrefs to the logo or author links.
+    a[href*="?authors"]:after,
+    a.m-info-unit_heading-link:after,
+    .m-info-unit_content h4 a:after {
+        // Don't append hrefs to the logo, author links or info unit headings.
```

Comment:
Should we be any more specific with the logo one That one is `a[href="/"]:after`, right? Which would be any link to the homepage, not necessarily just the logo, right?

Reply:
👍  I'll swap it out for `.o-header_logo-img`.